packages/wallet/backend/migrations/20240903125600_update_wallet_addresses.js

```diff
@@ -0,0 +1,44 @@
+const Knex = require('knex')
+const knexConfig = require('../knexfile')
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = async function (knex) {
+  const walletAddresses = await knex('walletAddresses').select(
+    'id',
+    'accountId'
+  )
+  for (const walletAddress of walletAddresses) {
+    const account = await knex('accounts')
+      .where('id', walletAddress.accountId)
+      .first()
+    if (account) {
+      await knex('walletAddresses')
+        .where('id', walletAddress.id)
+        .update({ assetCode: account.assetCode })
+    }
+  }
```

Comment:
Why is this migration needed? Wouldn't an already existing wallet address have the same asset code as its account?

Reply:
Looking into the db, I saw that WAs have a null asset code, except for those that are WM enabled. This migration is only for correcting that and making all the WAs have their respective asset code.

Reply:
Looking at the migration history is looks like it was introduced in `20231018084125_update_payment_pointers_table.js`, when the `isWM` flag was added as well. Not sure if this column is still needed?

CC @dragosp1011 @beniaminmunteanu

Reply:
yes it makes sense for this to be added due to the introduction of the WM payment pointers. 
That would be the only case I think of that wallet addresses would have different assets than the accounts that hold those WA.

Not needed anymore IMO.